starport/templates/app/stargate/readme.md.plush

```diff
@@ -30,6 +29,23 @@ npm run serve
 
 The frontend app is built using the `@starport/vue` and `@starport/vuex` packages. For details, see the [monorepo for Starport front-end development](https://github.com/tendermint/vue).
 
+## Release
+To release a new version of your blockchain, create and push a new tag with `v` prefix. A new draft release with the configured targets will be created.
+
+```
+git tag v0.1
+git push origin v0.1
+```
+
+After a draft release is created, make your final changes from the release page and publish it.
+
+### Install
+To install the latest version of your blockchain node's binary, execute the following command on your machine:
+
+```
+curl https://get.starport.network/<%= OwnerAndRepoName %>! | sudo bash
+```
+
```

Comment:
```suggestion
`<%= OwnerAndRepoName %>` should match the `username` and `repo_name` of the Gtihub repository to which the source code was pushed. Learn more about [the install process](https://github.com/allinbits/starport-installer).
```

Added this bit, because if a user pushes the code to a repo that doesn't match Go module path, they will not be able to install. Sounds obvious, but what if it isn't.